test/e2e/multikueue/e2e_test.go

```diff
@@ -201,6 +199,11 @@ var _ = ginkgo.Describe("MultiKueue", func() {
 
 			ginkgo.By("Creating the job", func() {
 				gomega.Expect(k8sManagerClient.Create(ctx, job)).Should(gomega.Succeed())
+				gomega.Eventually(func(g gomega.Gomega) {
+					createdJob := &batchv1.Job{}
+					g.Expect(k8sManagerClient.Get(ctx, client.ObjectKeyFromObject(job), createdJob)).To(gomega.Succeed())
+					g.Expect(ptr.Deref(createdJob.Spec.ManagedBy, "")).To(gomega.BeEquivalentTo(kueuealpha.MultiKueueControllerName))
```

Comment:
```suggestion
					g.Expect(createdJob.Spec.ManagedBy).To(gomega.BeEquivalentTo(ptr.To(kueuealpha.MultiKueueControllerName)))
```
I think this would read better.
```

Reply:
For pointers Equals will not work, but BeEquivalentTo, or BeComparableTo would work. 

Anyway, it is as nit, I'm ok to keep as is, just pointing out that the additional defaulting makes it tricky to read.